src/vivarium_cluster_tools/psimulate/branches.py

```diff
@@ -171,17 +175,16 @@ def calculate_random_seeds(
     if not random_seed_count:
         return []
 
-    np.random.seed(654321)
+    MAX_SEED_COUNT = 10000
+    if random_seed_count > MAX_SEED_COUNT:
+        raise ValueError(f"Random seed count must be less than {MAX_SEED_COUNT}.")
 
+    np.random.seed(654321)
+    possible = list(range(MAX_SEED_COUNT))
     if existing_seeds:
-        min_possible = max(existing_seeds) + 1
-    else:
-        existing_seeds = []
-        min_possible = 0
-
-    low, high = min_possible, min_possible + 10 * random_seed_count
-    possible = list(range(low, high))
-    return np.random.choice(possible, random_seed_count, replace=False).tolist()
+        possible = list(set(possible).difference(existing_seeds))
```

Comment:
Same as above about sets vs lists.

Reply:
I'm going to keep it as sets, with the sorted call. In terms of performance, there didn't seem to be a significant difference.